### PR TITLE
Prettier ignore inscription.ts file

### PR DESCRIPTION
**Pull request by ivy-lli**

### CODE REVIEW

File: packages/protocol/src/data/inscription.ts [hunk 1]
```diff
@@ -1,11 +1,12 @@
 /* eslint-disable */
+// prettier-ignore
```
Comment (ivy-lli) [inline packages/protocol/src/data/inscription.ts:2]: This line will disable prettier for this file

**Comment by ivy-lli** [comment]
@ivy-rew Somehow the ordering of the schema seems not to be the same after every change...

**Review by ivy-rew — COMMENTED** [review]
:+1:

**Comment by ivy-rew** [comment]
> @ivy-rew Somehow the ordering of the schema seems not to be the same after every change...

yep, I don't like that either. In theory I can enforce an order in the raw schema file ... let's make a task to address if it really hurts.

**Pull request merged**